README.md

```diff
@@ -51,7 +51,7 @@ After your pull request was merged, I will take care of regularly creating updat
 
 ## :newspaper: Academic & Press Recognition
 
-<img src="https://github.com/louis-e/arnis/blob/main/gitassets/recognition.png?raw=true" width="100%" alt="Banner">
+<img src="https://github.com/louis-e/arnis/blob/main/assets/gitassets/recognition.png?raw=true" width="100%" alt="Banner">
```

Comment:
FYI, we can link to images through relative paths in GitHub: https://stackoverflow.com/questions/15764242/is-it-possible-to-make-relative-link-to-image-in-a-markdown-file-in-a-gist#36313391

I think this provides the following advantages:
- When changing assets in a branch, they show up properly in that README (instead of always linking to the main branch)
- Assets will load locally when viewing the markdown without needing to call out to GitHub (useful when not having internet access, or when GitHub is down, since it goes down pretty often)